ansible-playbook-base/roles/configure-gluster-volume/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+# Tasks sets specification for gluster volume
+
+- name: Include gluster vol specification details
+  include_vars:
+    file: "{{ vs_definition }}"
+  run_once: true
+
```

Comment:
Add a play to rest the existing options to defaults, before setting the new options.